tests/rptest/tests/redpanda_test.py

```diff
@@ -28,17 +28,19 @@ def __init__(self,
                  extra_rp_conf=dict(),
                  enable_pp=False,
                  enable_sr=False,
-                 resource_settings=None,
-                 si_settings=None):
+                 **kwargs):
+        """
+        Any trailing keyword arguments are passed through to the
+        RedpandaService constructor.
+        """
         super(RedpandaTest, self).__init__(test_context)
         self.scale = Scale(test_context)
         self.redpanda = RedpandaService(test_context,
                                         num_brokers,
                                         extra_rp_conf=extra_rp_conf,
                                         enable_pp=enable_pp,
                                         enable_sr=enable_sr,
-                                        resource_settings=resource_settings,
-                                        si_settings=si_settings)
+                                        **kwargs)
```

Comment:
we'll lose a lot of benefits like: type checking (I think), IDE autocompletion, finding call sites when refactoring. building types like `ResourceSettings` seems like a good compromise. but I dunno maybe those are minor in comparison to the benefits?

EDIT: nevermind I wasn't paying attention and I thought this was changing the interface to RedpandaService. I *think* pyright can analyze the kwargs being passed through.